### ISSUE TITLE
Suggestion: use statics instead of free-range globals

### ISSUE THREAD

**Issue by SuperWomble** [Dimillian/MovieSwiftUI#9]
Firstly, this is a very nicely written project. The code is clear, well-written. Thank you for sharing the project with us.

However, I was confused by some free-roaming globals (e.g. '`store`', ''`sampleMovie`'). These are very hard to follow, particularly when not using an IDE (e.g. reading code on GitHub). I spent time looking for subclass properties, local vars, extensions etc.

Suggestion: add such properties and functions as static members on the relevant types. This provides context at the call site.

e.g. 

```
struct Movie: Codable, Identifiable {
    static let sampleMovie = Movie(...)
    ...
}

print(Movie.sampleMovie.id)

```

Cheers.

**Comment by SuperWomble** [comment]
Thirdly, please consider using 'private' or 'fileprivate' wherever possible. This greatly helps code comprehension.

For example, in `MovieDetail.swift`, the `fetchMovieDetails` function and the `addActionSheet` property can be made `private`.  Doing so makes their scope (and thus their ownership and limitations) clearly visible.

Cheers.

**Comment by Dimillian** [comment]
I've refactored a lot fo code in order to have correct access control, and it'll better over time :)
